system/indy-node-tests/test_vc.py

```diff
@@ -12,45 +12,42 @@
 # logger = logging.getLogger(__name__)
 # logging.basicConfig(level=0, format='%(asctime)s %(message)s')
 
+
 @pytest.fixture(scope='function', autouse=True)
 @async_generator
 async def docker_setup_and_teardown(nodes_num):
     await setup_and_teardown(nodes_num)
 
 
 @pytest.mark.asyncio
-async def test_vc_by_restart(pool_handler, wallet_handler, get_default_trustee):
+async def test_vc_by_restart(
+        pool_handler, wallet_handler, get_default_trustee, nodes_num
```

Comment:
You might consider to use `check_no_failures_fixture` to monitor node services failures to interrupt test in earlier phase if they happen.

Reply:
Done.